Allow ipa-certupdate to force specific server

KoRORland opened the pull request:
Adding option to specify a server to ipa-certupdate tool.
    
    Fixes: https://pagure.io/freeipa/issue/9839
    Signed-off-by: Aleksandr Sharov (asharov@redhat.com)

## Summary by Sourcery

Add a new --force-server option to ipa-certupdate to target a specific IPA server for certificate updates, update code to respect the forced server, and include documentation and tests to verify the new behavior.

New Features:
- Add --force-server option to specify server for certificate update in ipa-certupdate

Enhancements:
- Support variadic certificate tuple unpacking in update_db

CI:
- Update PRCI configuration to run test_commands suite and increase timeout

Documentation:
- Update ipa-certupdate man page with --force-server option and revise manual date

Tests:
- Add integration test to verify certupdate works with --force-server

rcritten commented on ipatests/test_integration/test_commands.py:
```diff
@@ -1471,6 +1471,35 @@ def test_certupdate_no_schema(self):
+        assert result.returncode != 0
+        assert "cannot connect to 'ldap://%s:389" % non_server \
+            in result.stderr_text
+
```
I'd also add a test for an existing server that is down. You can, for example, stop replicas[0] and re-run ipa-certupdate and it should fail gracefully.

KoRORland replied:
Done

KoRORland commented on ipaclient/install/ipa_certupdate.py:
```diff
@@ -75,15 +87,23 @@ def run(self):
 
 
-def run_with_args(api):
+def run_with_args(api, self):
```
fixed

flo-renaud commented on ipaclient/install/ipa_certupdate.py:
```diff
@@ -75,15 +87,24 @@ def run(self):
 
 
-def run_with_args(api):
+def run_with_args(api, options):
```
The command ipa-ca-install is internally calling `ipa_certupdate.run_with_args(api)` ([here](https://github.com/KoRORland/freeipa/blob/8a6ddeddc9926de9f37c7be873687e5fafe459b3/install/tools/ipa-ca-install.in#L192) and [there](https://github.com/KoRORland/freeipa/blob/8a6ddeddc9926de9f37c7be873687e5fafe459b3/install/tools/ipa-ca-install.in#L288C5-L288C38)) and it is missing the new `options` argument

KoRORland replied:
Not sure if current AzurePipeline errors are connected, doesn't seem so

flo-renaud commented:
Also IMO the option should be named --force-server to remain consistent with the changes done in https://github.com/freeipa/freeipa/pull/7687

KoRORland commented:
@flo-renaud I've changed naming and found error - I was still calling api.bootstrap() without passing the forced server as a parameter, so we were defaulting to the original one for api.Backend.rpcclient.connect(), but were running the ldapclient to a correct one. Fixed.
